addons/mail/static/src/core/common/mail_popout_service.js

```diff
@@ -47,7 +46,7 @@ export const mailPopoutService = {
          * @param {Boolean} triggerResize: Whether to trigger a resize event on the main window
          * @returns {Window} The external window
          */
-        function popout(element, triggerResize = true) {
+        function popout(app, toggleVisibility, triggerResize = true) {
```

Comment:
It should be component and props
```suggestion
        function popout(Component, props, toggleVisibility, triggerResize = true) {
```